test/e2e/cron/basic.yaml

```diff
@@ -3,7 +3,7 @@ kind: CronWorkflow
 metadata:
   name: test-cron-wf-basic
   labels:
-    argo-e2e: true
```

Comment:
I don't think any files in `test/e2e` should change - we actually to test that we support `argo-e2e: true` - because while it is not valid - we need to support it because of all the existing users - and having tests is how we ensure this

Reply:
Up to you, the test has a list of file/directory names to ignore so it'd be easy to ignore one example with the invalid setting and still require the rest (and future) to be valid per the schema unless added to the ignore list. There were a handful of other e2e tests that had missing `[]`/`{}` and missing keys that were required by the schema.

Reply:
I'm going to ask you to leave the E2E tests today

Reply:
done